lib/Settings/Admin.php

```diff
@@ -21,25 +21,31 @@
 
 namespace OCA\LogReader\Settings;
 
+use OCA\LogReader\Service\SettingsService;
 use OCP\AppFramework\Http\TemplateResponse;
-use OCP\IL10N;
+use OCP\AppFramework\Services\IInitialState;
 use OCP\Settings\IDelegatedSettings;
+use OCP\Util;
 
 class Admin implements IDelegatedSettings {
-	/** @var IL10N */
-	private $l;
+	private IInitialState $initialState;
+	private SettingsService $settingsService;
 
-	public function __construct(IL10N $l) {
-		$this->l = $l;
+	public function __construct(IInitialState $initialState, SettingsService $settingsService) {
+		$this->initialState = $initialState;
+		$this->settingsService = $settingsService;
```

Comment:
```suggestion
	public function __construct(
		private IInitialState $initialState, 
		private SettingsService $settingsService,
	) {
```
You missed this one no? Conversation is resolved but property promotion still not used.